trin-core/src/portalnet/overlay_service.rs

```diff
@@ -2054,9 +2054,10 @@ const TALK_REQ_PACKET_OVERHEAD: usize = 16 + // IV
     9 + // Request ID, max 8 bytes + 1 for RLP encoding
     3 + // RLP Encoding of inner response
     16; // RLP HMAC
-        // ENR SSZ overhead empirically observed to be double.
-        // Todo: determine why this is. It seems too high.
-const MAX_NODES_SIZE: usize = MAX_DISCV5_PACKET_SIZE - TALK_REQ_PACKET_OVERHEAD;
+const NODES_PACKET_OVERHEAD: usize = 1 + // Selector byte
+    1; // `total` field
```

Comment:
IIUC the overhead would not be the same for a `Content` and `Nodes` response because the `Content` message is a SSZ union, which has a `u32` type index. It seems like we only account for the `Nodes` message here.